src/lib/core/gestures/MdGestureConfig.ts

```diff
@@ -4,16 +4,25 @@ import {HammerGestureConfig} from '@angular/platform-browser';
 /* Adjusts configuration of our gesture library, Hammer. */
 @Injectable()
 export class MdGestureConfig extends HammerGestureConfig {
+  private _hammer = typeof window !== 'undefined' ? (window as any).Hammer : null;
```

Comment:
Right- instead of dropping all the way down to `any`, though, we can introduce out own interface and say `(window as any).Hammer as GestureManager`